app/assets/stylesheets/farmbot.css.scss

```diff
@@ -107,20 +113,60 @@ $brown:          #CA8;
   color: $white;
   font-size: 20px;
   text-align: center;
-
+  display: inline-block;
+  font-style: normal;
+  cursor: pointer;
   &:after {
     content: '\f067';
   }
 }
+
+  .farm-designer-body {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .farm-designer-left {
+    order: -1;
+    background-color: $light_green;
```

Comment:
Name of variable `light_green` should be written in all lowercase letters with hyphens instead of underscores